test/extended/workspacetype/workspace.go

```diff
@@ -61,4 +63,71 @@ var _ = g.Describe("[sig-workspace]", func() {
 			return workSpaces
 		}, 60*time.Second, 5*time.Second).ShouldNot(o.ContainSubstring(myWorkSpace.Name))
 	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-[Smoke] I can list child workspaces under parent workspace", func() {
+		g.By("# Create a test workspace as parent workspace")
+		k.SetupWorkSpace()
+		parentWorkSpace := k.WorkSpace()
+
+		g.By("# Create five child workspaces under the parent workspace")
+		childWorkSpaces := []exutil.WorkSpace{}
+		for i := 1; i <= 5; i++ {
+			g.By(fmt.Sprintf("# Creating child workspace no.%v", i))
+			k.SetupWorkSpaceWithSpecificPath(parentWorkSpace.ServerURL)
+			childWorkSpace := k.WorkSpace()
+			childWorkSpaces = append(childWorkSpaces, childWorkSpace)
+		}
+
+		g.By("# List child workspaces under the parent workspace, check if all workspaces are correctly displayed")
+		workSpaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ServerURL)
+		for i := 0; i < len(childWorkSpaces); i++ {
+			childWorkSpace := childWorkSpaces[i]
+			// NOTE: WorkSpace name inequality bug: https://url.corp.redhat.com/1878a60
+			var workSpace string
+			for j := 0; j < len(workSpaces); j++ {
+				workSpace = workSpaces[j]
+				if strings.Contains(workSpace, childWorkSpace.Name) {
+					break
+				}
+			}
+			o.Expect(workSpaces).To(o.ContainSubstring(childWorkSpace.Name))
+		}
+	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-[Smoke][Serial] I can create context for a specific workspace and use it", func() {
+		g.By("# Create a test workspace")
+		k.SetupWorkSpace()
+		workSpace := k.WorkSpace()
+
+		g.By("# Get context before test run")
+		contextBeforeTestRun, err := k.WithoutNamespace().WithoutKubeconf().Run("config").Args("current-context").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Create context under this workspace")
+		err = k.WithoutNamespace().WithoutKubeconf().Run("kcp").Args("workspace", "create-context", workSpace.Name, "--server="+workSpace.ParentServerURL).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Switch to the newly created context")
+		defer func() {
+			err = k.WithoutNamespace().WithoutKubeconf().Run("config").Args("use-context", contextBeforeTestRun).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			err = k.WithoutNamespace().WithoutKubeconf().Run("config").Args("delete-context", workSpace.Name).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
```suggestion
```
Seems xingxing suggested  to remove the assert to avoid unnecessary failed result if `delete-context` failed and it should be safe no influence other cases. I also suggest to remove it.